doc/source/docs/user_guide/projections.rst

```diff
@@ -121,18 +121,12 @@ projection passed to GeoSeries or GeoDataFrame during the creation:
    - Lat[north]: Geodetic latitude (degree)
    - Lon[east]: Geodetic longitude (degree)
    ...
-   >>> GeoSeries(array, crs=3395).crs  # crs=3395 is ignored as array already has CRS
-   FutureWarning: CRS mismatch between CRS of the passed geometries and 'crs'. Use 'GeoDataFrame.set_crs(crs, allow_override=True)' to overwrite CRS or 'GeoDataFrame.to_crs(crs)' to reproject geometries. CRS mismatch will raise an error in the future versions of GeoPandas.
+   >>> GeoSeries(array, crs=4326)  # crs=4326 is okay, as it matches the existing CRS
+   >>> GeoSeries(array, crs=3395)  # crs=3395 is forbidden as array already has CRS
+   ValueError: CRS mismatch between CRS of the passed geometries and 'crs'. Use 'GeoDataFrame.set_crs(crs, allow_override=True)' to overwrite CRS or 'GeoDataFrame.to_crs(crs)' to reproject geometries.
```

Comment:
This should be a GeoSeries error, not GeoDataFrame.

Reply:
Good catch.